client/app/bundles/course/video/submission/reducers/oldSessions.js

```diff
@@ -0,0 +1,37 @@
+import { List as makeImmutableList, Map as makeImmutableMap } from 'immutable';
+import { createTransform } from 'redux-persist';
+import { sessionActionTypes } from 'lib/constants/videoConstants';
+
+/**
+ * oldSessions state slice is an ImmutableMap mapping a old session id to a old video state slice.
+ *
+ * Refer to the video reducer for the format of the video state slice.
```

Comment:
Maybe can describe *when* the data here is updated/populated, since one might expect one of the reducer actions to do it. Maybe can just add a note to point to the stateReconciler docs.